starboard/android/shared/application_android.cc

```diff
@@ -209,19 +210,23 @@ bool ApplicationAndroid::DestroyWindow(SbWindow window) {
 
 Event* ApplicationAndroid::WaitForSystemEventWithTimeout(int64_t time) {
   // Limit the polling time in case some non-system event is injected.
-  const int kMaxPollingTimeMillisecond = 10;
+  const int kMaxPollingTimeMillisecond = 1000;
```

Comment:
I missed this change. Will it be too long to wait for 1 second to pull system events?
Will this drag down the performance of the app, and then cause more ANRs?